hazelcast/src/main/java/com/hazelcast/config/ReplicatedMapConfig.java

```diff
@@ -364,6 +392,9 @@ public void readData(ObjectDataInput in) throws IOException {
         statisticsEnabled = in.readBoolean();
         mergePolicy = in.readUTF();
         listenerConfigs = readNullableList(in);
+        if (in.getVersion().isGreaterOrEqual(Versions.V3_10)) {
```

Comment:
Yes. This guards is to avoid serialization errors. 
At the end I'm planning to add a cut-off point on the quorum service to skip quorum for the new structures when the cluster version is lower than 3.10